go.mod

```diff
@@ -63,7 +63,7 @@ require (
 	github.com/google/go-github/v57 v57.0.0
 	github.com/google/uuid v1.6.0
 	github.com/grafana-tools/sdk v0.0.0-20220919052116-6562121319fc
-	github.com/grafana/alerting v0.0.0-20240607182251-835aff588914
+	github.com/grafana/alerting v0.0.0-20240626080128-8299cb22b8df
```

Comment:
This comes from [here](https://github.com/grafana/alerting/commits/grobinson/fix-silence-fixes). It is a branch that has two commits on top of `835aff5` to avoid bringing in the changes from #8515 while it is still in review.

Reply:
@santihernandezc asked me why do this instead of using `main` from `grafana/alerting`. My reasoning for this was so I don't have to bring in changes from #8515 for the bug fix as I don't know if and when that PR will be approved. If you consider this to be unnecessary I will happily update the PR to use `main`.